pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -192,6 +192,13 @@ func SetDefaults_LeaderElectionConfiguration(obj *LeaderElectionConfiguration) {
 }
 
 func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
+	// pointer because the zeroDuration is valid - if you want to skip the trial period
+	if obj.ConfigTrialDuration == nil {
+		obj.ConfigTrialDuration = &metav1.Duration{Duration: 10 * time.Minute}
+	}
+	if obj.CrashLoopThreshold == nil {
+		obj.CrashLoopThreshold = utilpointer.Int32Ptr(3)
```

Comment:
Can we bump this up to 10 for now until we resolve https://github.com/kubernetes/kubernetes/issues/50216? Otherwise, this could be very dangerous for anyone attempt to use this feature end up to no good kubelet config.